### ISSUE TITLE
Valid phone numbers are not accepted

### ISSUE THREAD

**Issue by wichert** [ttacon/libphonenumber#13]
As an experiment I tried to validate ` +31101234567`. Testing this against the [libphonenumber demo](https://rawgit.com/googlei18n/libphonenumber/master/javascript/i18n/phonenumbers/demo-compiled.html) shows that it should be flagged as a correct number. When I run this against the golang version I get the following:

```
Number: country_code:31 national_number:1234567 
Region code: NL
Number is valid: false
```

```
	number, err := libphonenumber.Parse(phone, "MY")
	if err != nil {
		log.Printf("Invalid: %v", err.Error())
		return false
	}
	log.Printf("Number: %v", number)
	log.Printf("Region code: %v", libphonenumber.GetRegionCodeForNumber(number))
	log.Printf("Number is valid: %v", libphonenumber.IsValidNumber(number))
```

It looks like the area code (`010`) was lost somewhere.

**Comment by ttacon** [comment]
Ya, I currently think there might be a metadata issue that needs to be looked into (possibly related to #7 ). I don't have a ton of time at the moment (day job and all) but when I do I'm planning to get around to this.

**Comment by ttacon** [comment]
This has been fixed by #14
